sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -61,6 +63,64 @@ class SparkSqlParserSuite extends AnalysisTest {
   private def intercept(sqlCommand: String, messages: String*): Unit =
     interceptParseException(parser.parsePlan)(sqlCommand, messages: _*)
 
+  test("Checks if SET/RESET can accept all the configurations") {
+    // Force to build static SQL configurations
```

Comment:
How about spark conf ? Check this PR https://github.com/apache/spark/pull/23031/files

Reply:
Added tests for the Spark confs, too.